fairscale/nn/wrap/auto_wrap.py

```diff
@@ -4,23 +4,72 @@
 # LICENSE file in the root directory of this source tree.
 
 import contextlib
-from typing import Any, Callable, Dict, Generator, List, Optional, Tuple
+from typing import Any, Callable, Dict, Generator, Optional, Set, Tuple, cast
 
 import torch.nn as nn
 
-from fairscale.nn.data_parallel.fully_sharded_data_parallel import FullyShardedDataParallel
 from fairscale.nn.misc import checkpoint_wrapper
 
-# Modules that don't wrap.
-FSDP_MODULE_EXCLUDE_WRAP = {nn.ModuleList, nn.ModuleDict}
-# Modules that we don't recurse down to their children.
-FSDP_MODULE_BLOCKLIST = {nn.MultiheadAttention}
+
+def should_wrap_default(
```

Comment:
what do you think about renaming this slightly to make the intent clearer, e.g., `default_should_wrap_policy`?

Reply:
done.